src/containers/pages/InterventionPlan/IRS/plan/index.tsx

```diff
@@ -446,7 +449,11 @@ class IrsPlan extends React.Component<
     }
 
     // update state after fetching plan from OpenSRP
-    if (!newPlan && planById && planById.plan_jurisdictions_ids) {
+    // update state after saving plan to store
+    if (
+      (!newPlan && planById && planById.plan_jurisdictions_ids) ||
```

Comment:
you can simplify this to mention `planById` only once

Reply:
Done